app/controllers/users_controller.rb

```diff
@@ -44,7 +44,7 @@ def destroy
 
   private
   def user_params
-    params.require(:user).permit :name, :email, :password, :password_confirmation
+    params.require(:user).permit :name, :email, :is_admin, :password, :password_confirmation
```

Comment:
pleas use brackets with multiple attributes in `permit` method:
`params.require(:user).permit(:name, :email, :is_admin, :password, :password_confirmation)`